ingest/src/main/java/com/microsoft/azure/kusto/ingest/IngestionProperties.java

```diff
@@ -67,6 +86,19 @@ public void setAuthorizationContextToken(String token)
         additionalProperties.put("authorizationContext", token);
     }
 
+    /**
+     * Creates an initialized {@code IngestionProperties} instance with a given {@code databaseName} and {@code tableName}.
+     * The default values of the rest of the properties are:
+     * <blockquote><code>
+     *   <p>reportLevel = IngestionReportLevel.FailuresOnly;</p>
+     *   <p>reportMethod = IngestionReportMethod.Queue;</p>
+     *   <p>flushImmediately = false;</p>
+     *   <p>additionalProperties = new HashMap();</p>
+     * </code></blockquote>
+     *
+     * @param databaseName the name of the database in the destination Kusto cluster.
+     * @param tableName the name of the table in the destination database.
+     */
```

Comment:
please revise the hash map to avoid a code-check alret:
additionalProperties = new HashMap<>();

Also, maybe all lines should start with "this" to make it look uniform?

Reply:
changed